lottie/src/main/java/com/airbnb/lottie/model/layer/TextLayer.java

```diff
@@ -174,6 +176,13 @@ private void drawTextGlyphs(
     } else {
       textSize = documentData.size;
     }
+    String textStyle = null;
```

Comment:
You only included support for glyphs. Can you include the code path that draws text with a font as well?

Reply:
@gpeal I did, please check line 269